torch/distributions/wishart.py

```diff
@@ -20,15 +20,20 @@ def _mvdigamma(x: torch.Tensor, p: int) -> torch.Tensor:
         - torch.arange(p, dtype=x.dtype, device=x.device).div(2).expand(x.shape + (-1,))
     ).sum(-1)
 
+def _clamp_with_eps(x: torch.Tensor) -> torch.Tensor:
```

Comment:
related: https://github.com/pytorch/pytorch/issues/41492, this might not be torchscriptable...

Reply:
Thank you for letting me know. I will try to find alternatives. It is hard to judge which function is torchscriptable.

Reply:
Also feel free to chime in that issue, maybe its priority might be increased

Reply:
I don't think I understand the mechanism of `torch.script` yet, but I will try on. Thanks for the suggestion!

Reply:
Thanks for suggestion. I will modify the name.